datafusion-examples/examples/plan_to_sql.rs

```diff
@@ -52,7 +52,7 @@ fn simple_expr_to_sql_demo() -> Result<()> {
     let expr = col("a").lt(lit(5)).or(col("a").eq(lit(8)));
     let ast = expr_to_sql(&expr)?;
     let sql = format!("{}", ast);
-    assert_eq!(sql, r#"(("a" < 5) OR ("a" = 8))"#);
+    assert_eq!(sql, r#"((a < 5) OR (a = 8))"#);
```

Comment:
❤️

Reply:
Given this change, perhaps we can remove the next example in the file `simple_expr_to_sql_demo_no_escape` as I don't think it serves any purpose